### ISSUE TITLE
GOGGalaxy: Cannot install games which require .NET redist

### ISSUE THREAD

**Issue by TyPh00nCdr** [Ash258/Scoop-Ash258#176]
Divinity Original Sin 2 will not install with GOGGalaxy if installed with scoop (neither with a local, nor a global install). If installed independently from the GOGGalaxy's website, I did not have any problems.

It seems the game requires .NET 3.5 to be installed and it can't check for the current .NET install or install .NET, respectively, if GOGGalaxy was installed with scoop. After I had uninstalled GOGGalaxy with scoop and had installed it with the website's installer, .NET 3.5 was installed successfully with the game's installation and I could launch the game. Investigating the GameClient-logs from GOGGalaxy also revealed it repeatedly failed to install the game's redistributable.

**Comment by Ash258** [comment]
And??

Just install .net35

**Comment by TyPh00nCdr** [comment]
First of all, I only didn't know the reason the install failed was .NET. GOGGalaxy just failed the game's installation. I only found out, when the .NET-installer popped up after I had removed the scoop installation of Galaxy and installed it with the web installer. How do I find out which dependency is missing next time a game installation fails?

Secondly, even after .NET was installed and I reinstalled Galaxy with scoop, the game still couldn't be launched, because Galaxy still couldn't verify all dependencies were installed correctly. Again, this only worked with the standalone installed Galaxy.

tl;dr: It's impossible to play Divinity Original Sin 2 with the scoop installation of GOGGalaxy and it has probably something to do with GOGGalaxy's path or permissions.
